### ISSUE TITLE
预训练模型性能问题

### ISSUE THREAD

**Issue by AIChuY** [Henry1iu/TNT-Trajectory-Prediction#31]
您好！我现在在试图用您提供的预训练权重重现模型，但是其表现似乎有些问题。在您提供的小型数据集的验证集上测试
python3 test_tnt.py -rm /media/wzy/78728a5f-27bc-43c6-ae04-0d7615fc9e86/wzy/TNT/TNT/best_TNT.pth -r ../interm_data_small -s val
得到的结果很差
The test result: {'minADE': 8.633853370351169, 'minFDE': 16.05028510884763, 'MR': 0.7764705882352941}
我自己作了可视化之后发现确实有一些很明显的错误
![image](https://user-images.githubusercontent.com/60698001/189284718-f77f5b8f-aa9b-46fe-9d4b-a007afdf7635.png)
不知道您有没有头绪可能是什么造成这个结果？

**Comment by AIChuY** [comment]
似乎是包版本不一致导致的问题，在重新配置环境之后表现正常了
![image](https://user-images.githubusercontent.com/60698001/189290902-bd9f67a8-ca98-49b6-9d58-5daff67835b9.png)

**Comment by MozhganPourKeshavarz** [comment]
Hi @AIChuY 
would you explain what was the problem? and have you tried an evaluation with the pre-trained weights on the actual test set?
